lib/foundation_rails_helper/form_builder.rb

```diff
@@ -129,6 +129,42 @@ def custom_label(attribute, text, options, &block)
       label(attribute, text, options)
     end
 
+    def tag_from_options(name, options)
+      if options && options[:size].present? && options[:value].present?
+        content_tag(:div,
+                    content_tag(:span, options[:value], :class=>name),
+                    :class=>"small-#{options[:size]} large-#{options[:size]} columns")
```

Comment:
It would be more beneficial to be able to have a different sizes for small, medium, and large columns.  If you are using the same size for small and large, you wouldn't need to set `small-2 large-2`, you could just use `small-2` for the class.  But, I often have to do different column sizes for each break-point.